### PR TITLE
Rules should be defined a dotnet-project-file-analyzers.github.io

### PR DESCRIPTION
**Pull request by Corniel**
For cleaner URL's all rules defined at [dotnet-project-file-analyzers.github.io](https://dotnet-project-file-analyzers.github.io/). To prevent inconsistencies, rules should not be documented both at this GitHub repo and at dotnet-project-file-analyzers.github.io.

### CODE REVIEW

File: specs/DotNetProjectFile.Analyzers.Specs/Design_specs.cs [hunk 1]
```diff
@@ -8,21 +9,35 @@ public class Rules
     public void Ids_are_unique()
         => Descriptors.Select(d => d.Id).Should().OnlyHaveUniqueItems();
 
-    [TestCaseSource(nameof(RuleIds))]
-    public void have_mark_down_documentation(string id)
-        => new FileInfo($"../../../../../rules/{id}.md").Exists.Should().BeTrue(because: $"{id}.md should exist.");
+    /// <remarks>
+    /// To
```
Comment (CptWesley) [inline specs/DotNetProjectFile.Analyzers.Specs/Design_specs.cs:13]: unfinished comment

File: specs/DotNetProjectFile.Analyzers.Specs/Design_specs.cs [hunk 2]
```diff
@@ -44,18 +59,33 @@ private static IEnumerable<Type> Types
         .GetTypes()
         .Where(t => !t.IsAbstract && t.IsAssignableTo(typeof(DiagnosticAnalyzer)));
 
-    private static IEnumerable<DiagnosticDescriptor> Descriptors
+    private static IEnumerable<Rule> Descriptors
```
Comment (CptWesley) [inline specs/DotNetProjectFile.Analyzers.Specs/Design_specs.cs:62]: I think `Rules` might now be a more descriptive name.

File: specs/DotNetProjectFile.Analyzers.Specs/Design_specs.cs (hunk 1 above)
Comment (Corniel) [reply specs/DotNetProjectFile.Analyzers.Specs/Design_specs.cs]: Has been dropped.

File: specs/DotNetProjectFile.Analyzers.Specs/Design_specs.cs (hunk 2 above)
Comment (Corniel) [reply specs/DotNetProjectFile.Analyzers.Specs/Design_specs.cs]: Rules is not allowed, that's the name of the parent class, but `AllRules` is also better.

**Pull request merged**